contrib/cirrus/pr-should-link-jira

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Intended for use in CI, for -rhel branches: check git PR, barf if no jira links
+#
+
+ME=$(basename $0)
+
+set -e
+
+# Github label which allows overriding this check
+OVERRIDE_LABEL="No Jira Link"
+
+# Only -rhel branches need jira links
+BRANCH_REGEX="v.*-rhel"
+
+LINK_REGEX="Fixes:? [\\t ]*https:\\/\\/issues.redhat.com\\/[\w\\/-]*"
+
+if [[ ! "${DEST_BRANCH}" =~ $BRANCH_REGEX ]]; then
+    exit 0
+fi
+
+if [[ "${CIRRUS_CHANGE_MESSAGE}" =~ $LINK_REGEX ]]; then
+    exit 0
+fi
+
+# Nope. Only allow if the github 'No Jira Link' label is set
+if [[ -z "$CIRRUS_PR" ]]; then
+    echo "$ME: cannot query github: \$CIRRUS_PR is undefined" >&2
+    exit 1
+fi
+
+if [[ -z "$CIRRUS_REPO_CLONE_TOKEN" ]]; then
+    echo "$ME: cannot query github: \$CIRRUS_REPO_CLONE_TOKEN is undefined" >&2
+    exit 1
+fi
+
+query="{
+  \"query\": \"query {
+  repository(owner: \\\"containers\\\", name: \\\"podman\\\") {
+    pullRequest(number: $CIRRUS_PR) {
+      labels(first: 100) {
+        nodes {
+          name
+        }
+      }
+    }
+  }
+}\"
+}"
+
+result=$(curl -s -H "Authorization: bearer $CIRRUS_REPO_CLONE_TOKEN" -H "Accept: application/vnd.github.antiope-preview+json" -H "Content-Type: application/json" -X POST --data @- https://api.github.com/graphql <<<"$query")
+
+labels=$(jq -r '.data.repository.pullRequest.labels.nodes[].name' <<<"$result")
```

Comment:
I'm not certain about the github API behavior, what happens if there are no labels at all?  Consider if something like this might be more robust:

```suggestion
labels=$(jq -r '.data.repository.pullRequest.labels.nodes[]?.name' <<<"$result")
```

Reply:
This code was also copied from pr-should-include-tests, so I think it works. But I can change this if you want.

Reply:
I changed it